README.md

```diff
@@ -1,50 +1,60 @@
-# Doctor Droid - Java SDK for custom events
-This is a library that will allow developers to push custom stateful events to Doctor Droid Platform.
+# Doctor Droid - Java SDK for custom ingestionEvents
+
+This is a library that will allow developers to push custom stateful ingestionEvents to Doctor Droid Platform.
 Read more [here](https://kenobi.drdroid.io/docs).
 
 ## Install the SDK
+
 ### Compile SDK locally to your gradle project
-#### Add the following to your settings.gradle:
-```agsl
-include ':drdroidapi-java'
-project(':drdroidapi-java').projectDir = new File('<path to lib>/drdroidapi-java')
-```
 
 #### Add the following to your build.gradle:
+
 ```agsl
-implementation project(":drdroidapi-java")
+implementation group: 'io.drdroid', name: 'api-java', version: '1.0.0'
```

Comment:
@droid-mohit to update this to 2.0.0 once maven central sync is complete.

Reply:
Already change it. Maven central sync will happen in some time.

Reply:
Ack, updated